### ISSUE TITLE
AFB forgetting balloon convert

### ISSUE THREAD

**Issue by NoutEditz** [NatroTeam/NatroMacro#782]
### Have you searched the existing bug reports?

Yes

### What happened?

Pine x4 was getting low, so macro interrupted balloon convert to go to blue field booster (it wasnt ready yet), after that it started roling dice, it lost x4 by that time and it got dandelion and kept it. 
Everytime it now goes to convert it either just doesnt do it, or the timing with AFB is bad and starts roling dice and or going to blue field booster and forgetting about the balloon.
Also in RDP if its at hive the CONVERT HONEY bar will cover the buffs and the macro cant see which field its getting (if that is how it works)

In short Macro wants keep up pine x4, messes up, losing balloon blessing

### Steps to reproduce

1.Macro with afb (Dice, glitter and blue field booster) So far only know that it happends in an rdp
2.let it run a bit, and watch
3.Reset to convert, it will convert a bit and forget about balloon to go to field booster

### What would you expect to happen?

You would see the macro forgetting about the balloon and either going to blue field booster, or roll dice and go to the pine field forgetting balloon.

### Natro Macro version

1.0.0.2

### What is your Discord username?

porsche911rwb
